### PR TITLE
add README to document the qr image and its build and testing process

### PR DESCRIPTION
**Pull request by jonmosco**
Debugging the q-r image led me down a dependency path so I figured I would capture it for posterity.  

One additional minor change in file comments.

### CODE REVIEW

**Review by OliviaHY — APPROVED** [review]
❤️

File: dockerfiles/README.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+The [base image](https://github.com/app-sre/container-images/tree/master/qontract-reconcile-base) `qontract-reconcile-base` is used.
+
+This stage copies the `/work` directory from the `build-image` stage.
+
+#### Stage 4 - fips-prod-image
+
+The base image `prod-image` is used.
+
+This stage uses the external image `qontract-reconcile-oc`
```
Comment (cubismod) [inline dockerfiles/README.md:45]:
```suggestion
This stage uses the external image `qontract-reconcile-oc` to copy a specific `oc` version into the Qontract Reconcile image for use in FIPS environments.
```

File: dockerfiles/README.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+```
+
+`build-test` uses the image `qontract-reconcile-builder`, installs the necessary packages
+and runs the tox tests
+
+```Makefile
+build-test:
+	@$(CONTAINER_ENGINE) build -t $(IMAGE_TEST) -f dockerfiles/Dockerfile.test .
+```
+```
```
Comment (cubismod) [inline dockerfiles/README.md:116]:
```suggestion

```

**Review by cubismod — APPROVED** [review]
LGTM!

**Review by esron — COMMENTED** [review]
LGTM

**Pull request merged**